packages/circuit-ui/components/Modal/Modal.module.css

```diff
@@ -1,11 +1,28 @@
-.base {
+.dialog {
```

Comment:
The scoped class names are generated using the filename, class name and a hash. The `.base` class name is special in that it is omitted from the generated class name to avoid duplication (e.g. `cui-badge-badge`). Personally, I'd prefer to keep using `.base` instead of `.dialog` here (`cui-modal-dialog` → `cui-modal`).